haskell/app/Main.hs

```diff
@@ -30,6 +35,11 @@ main = wrapMain $ do
     PrintMetadata mc ->
       putTextLn . decodeUtf8 . encodePretty $
         mkSegCfmmMetadata (mkMetadataSettings mc)
+    PrintMetadataByte mc ->
+      putTextLn . encodeHex . BSL.toStrict . encode $
+        mkSegCfmmMetadata (mkMetadataSettings mc)
+    GenerateMetadataUri key ->
+      putStr . TZIP.encodeURI $ TZIP.tezosStorageUri TZIP.selfHost key
```

Comment:
shoudln't this use `encodeHex` too? Otherwise we'll just get a literal string in the output file

Reply:
Updated this.